doc/api/buffer.md

```diff
@@ -508,7 +508,7 @@ added: v5.10.0
 -->
 
 * `size` {Integer} The desired length of the new `Buffer`
-* `fill` {string | Buffer | Integer} A value to pre-fill the new `Buffer` with.
+* `fill` {string|Buffer|Integer} A value to pre-fill the new `Buffer` with.
```

Comment:
I think I'll move it out of `jsPrimitives` and into the `typeMap`, possibly linking it to `number` docs.